diva-irma.js

```diff
@@ -1,10 +1,14 @@
 /*!
- * diva-irma-js
+ * diva-irma
```

Comment:
since the package name is diva-irma-js, I think this should still be diva-irma-js